_benchmark/user-guide/running-workloads.md

```diff
@@ -0,0 +1,175 @@
+|                                              Median Throughput |     percolator_with_content_president_bush |       36.09 |  ops/s |
+|                                                 Max Throughput |     percolator_with_content_president_bush |       36.09 |  ops/s |
+|                                       100th percentile latency |     percolator_with_content_president_bush |     35.9822 |     ms |
+|                                  100th percentile service time |     percolator_with_content_president_bush |     7.93048 |     ms |
+|                                                     error rate |     percolator_with_content_president_bush |           0 |      % |
+
+[...]
+
+|                                                 Min Throughput |          percolator_with_content_ignore_me |        16.1 |  ops/s |
+|                                                Mean Throughput |          percolator_with_content_ignore_me |        16.1 |  ops/s |
+|                                              Median Throughput |          percolator_with_content_ignore_me |        16.1 |  ops/s |
+|                                                 Max Throughput |          percolator_with_content_ignore_me |        16.1 |  ops/s |
+|                                       100th percentile latency |          percolator_with_content_ignore_me |     131.798 |     ms |
+|                                  100th percentile service time |          percolator_with_content_ignore_me |     69.5237 |     ms |
+|                                                     error rate |          percolator_with_content_ignore_me |           0 |      % |
+|                                                 Min Throughput | percolator_no_score_with_content_ignore_me |       29.37 |  ops/s |
+|                                                Mean Throughput | percolator_no_score_with_content_ignore_me |       29.37 |  ops/s |
+|                                              Median Throughput | percolator_no_score_with_content_ignore_me |       29.37 |  ops/s |
+|                                                 Max Throughput | percolator_no_score_with_content_ignore_me |       29.37 |  ops/s |
+|                                       100th percentile latency | percolator_no_score_with_content_ignore_me |     45.5703 |     ms |
+|                                  100th percentile service time | percolator_no_score_with_content_ignore_me |      11.316 |     ms |
+|                                                     error rate | percolator_no_score_with_content_ignore_me |           0 |      % |
+
+
+
+--------------------------------
+[INFO] SUCCESS (took 18 seconds)
+--------------------------------
+```
+
+
+
+## Running a workload on your own cluster
+
+Now that you're familiar with running OpenSearch Benchmark on a cluster, you can run OpenSearch Benchmark on your own cluster, using the same `execute-test` command, replacing the following settings.
```

Comment:
Maybe update `Now that you're familiar with running OpenSearch Benchmark on a cluster, you can run OpenSearch Benchmark on your own cluster` to say `Now that you're familiar with running OpenSearch Benchmark on a local cluster, you can run OpenSearch Benchmark on your external cluster`